File: Java Programing Basics/ConditionalStatements(EXERCISE)/src/Scholarship.java
import java.util.Scanner;

public class Scholarship {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double MoneyGetsBGN = Double.parseDouble(scanner.nextLine());
        double MiddlePoints = Double.parseDouble(scanner.nextLine());
        double MiniMoney = Double.parseDouble(scanner.nextLine());
        double MidPoints = (35 / 100.0) * MiniMoney;
        double MidPointsToZero = Math.floor(MidPoints);
        double ExcelentPoints = MiddlePoints * 25;
        double ExcelentToZero = Math.floor(ExcelentPoints);

        if (5.50 > MiddlePoints ){
            System.out.println("You cannot get a scholarship!");
        }else if (MiddlePoints >= 5.50 && MoneyGetsBGN < MiniMoney){
            System.out.printf("You get a Social scholarship %.0f BGN", MidPointsToZero);
        }else
            System.out.printf("You get a scholarship for excellent results %.0f BGN", ExcelentToZero);

    }

}

File: Python/Conditional_Statements(EXERCISE)/Scholarship.py
import math

money_FamilyGet = float(input())
grade = float(input())
minimum_salary = float(input())

#Social Stipendi
if money_FamilyGet < minimum_salary and 4.5 <= grade < 5.50:
    minimum_salary = 35 / 100 * minimum_salary
    floored_minimum_salary = math.floor(minimum_salary)
    print(f"You get a Social scholarship {minimum_salary} BGN")

#Excellent Results
if grade >= 5.5 and money_FamilyGet > minimum_salary:
    grade *= 25
    floored_grade = math.floor(grade)
    print(f"You get a scholarship for excellent results {floored_grade} BGN")

#No scholarship
if grade < 5.5 and money_FamilyGet > minimum_salary:
    print("You cannot get a scholarship!")


#Full
if money_FamilyGet < minimum_salary and grade >= 5.5:

    grade *= 25
    minimum_salary = 35 / 100 * minimum_salary
    floored_minimum_salary = math.floor(minimum_salary)
    floored_grade = math.floor(grade)

    if grade < minimum_salary:
        print(f"You get a Social scholarship {floored_minimum_salary} BGN")

    if minimum_salary < grade:
        print(f"You get a scholarship for excellent results {floored_grade} BGN")

    if floored_minimum_salary == floored_grade:
        print(f"You get a scholarship for excellent results {floored_grade} BGN")

